rendered.yml

```
@@ -17,3 +17,4 @@ default_context:
   minimum_python_version: "3.6"
   required_dependencies: "astropy"
   optional_dependencies: ""
+  private_project: "y"
```

Would this not be better to render the default as no?

```suggestion
  private_project: "n"
```

---

Agreed. I think that was for a test I did.